### PR TITLE
Make email field as editable in edit profile page

### PR DESCRIPTION
**Pull request by techaffinity**
Now We can edit the email address in edit profile page.

### CODE REVIEW

File: project/templates/app/mobi/app/editprofile.html [hunk 1]
```diff
@@ -23,8 +23,10 @@
         {% endfor %}
     {% endif %}
   {% endif %}
-
-  {{ form.username }}
+  <label for="user_name" class="label-wide">{{ form.username.label_tag }}
+    {{ form.username }}
+  </label>
+  {{ form.email }}
```
Comment (miltontony) [inline project/templates/app/mobi/app/editprofile.html:29]: Won't this result in the email appearing twice?

**Comment by miltontony** [comment]
Please add a test for "A user with that email already exists"

File: project/templates/app/mobi/app/editprofile.html (hunk 1 above)
Comment (techaffinity) [inline project/templates/app/mobi/app/editprofile.html:29]: No, It won`t appear because the {{form.email}} used as hidden field.

**Comment by techaffinity** [comment]
The test case has been added for the user email check.

**Comment by miltontony** [comment]
:+1:

**Pull request merged**